docs/reference-guides/data/data-core.md

```diff
@@ -18,14 +18,14 @@ Calling this may trigger an OPTIONS request to the REST API via the
 
 _Parameters_
 
--   _state_ `Object`: Data state.
+-   _state_ `State`: Data state.
 -   _action_ `string`: Action to check. One of: 'create', 'read', 'update', 'delete'.
 -   _resource_ `string`: REST resource to check, e.g. 'media' or 'posts'.
--   _id_ `string=`: Optional ID of the rest resource to check.
+-   _id_ `number | string`: Optional ID of the rest resource to check.
```

Comment:
For consistency should we use `RecordKey` here too, and in the other functions expecting a resource `id`?